src/modules/Alert.jsx

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable prettier/prettier */
+/* eslint-disable react/prop-types */
```

Comment:
not a good idea to disable prop-types when you write in javascript. I'd suggest to get used to Proptypes + JS or use typescript

Reply:
thanks, I'll keep it in mind